drf_generators/management/commands/generate.py

```diff
@@ -45,15 +45,16 @@ def handle_app_config(self, app_config, **options):
             views = options['views'] if 'views' in options else False
             urls = options['urls'] if 'urls' in options else False
 
-        elif django.VERSION[1] >= 8:
+        elif django.VERSION[1] >= 8 or (django.VERSION[0] == 2
+                                        and django.VERSION[1] >= 0):
```

Comment:
This is failing the style check, `and` operator should be on the previous line

Reply:
My linter give me a warn if i put it on the previous line

    W504 line break after binary operator